pkg/scorecard.go

```diff
@@ -100,8 +121,18 @@ func RunScorecard(ctx context.Context,
 
 	commitSHA, err := getRepoCommitHash(repoClient)
 	if err != nil || commitSHA == "" {
+		//nolint:wrapcheck
```

Comment:
my understanding was the wrapcheck only complained about external errors?

Reply:
done.